### DISCUSSION TITLE
SSHOperator cmd_timeout

### DISCUSSION THREAD

**Discussion by punx120** [apache/airflow#27170] [Q&A]
Hi,

I just started using AirFlow with SSHOperator - the doc mentioned that we should be using cmd_timeout, but it doesn't seem to work for me. I looked at the code, and as far as i can tell it doesn't seem to do anything. Am I missing something?

Thank you

**Comment by o-nikolas** [comment]
Hey @punx120 

Thanks for taking the time to open this Discussion! I had a quick look and I think you're right about this.

The cmd_timeout was added in this PR: #17236

Then it was dropped from one of it's usages (it seems accidentally) in this PR: #17378
And the second was dropped in this PR (again, it seems accidental): #21326

I'm going to convert this to Github Issue :smiley:

**Reply by o-nikolas** [reply]
Issue was already created, can be found here: #27182
